inc/class-statify-frontend.php

```diff
@@ -91,15 +91,24 @@ public static function track_visit( $is_snippet = false ) {
 			$data['referrer'] = esc_url_raw( $referrer, array( 'http', 'https' ) );
 		}
 
-		/* Relative target url */
+		// Relative target URL.
 		$data['target'] = user_trailingslashit( str_replace( home_url( '/', 'relative' ), '/', $target ) );
 
-		// Trim target url.
+		// Trim target URL, i.e. remove query parameters.
 		if ( $wp_rewrite->permalink_structure ) {
-			$data['target'] = wp_parse_url( $data['target'], PHP_URL_PATH );
+			$parsed_target  = wp_parse_url( $data['target'] );
+			$data['target'] = $parsed_target['path'];
```

Comment:
can it be possible that `$parsed_target['path']` is not set?

Reply:
Good catch, we should add validation and pass `null` if not filled (at least that's what `wp_parse_url('https://example.com', PHP_URL_PATH)`  would have returned)